src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script
+      type="text/javascript"
+      src="./main.js">
+    </script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="formfild">
```

Comment:
Plese fix this class name to fieldset for example. Because fieldset it's group of inputs and logically more correct name it like fieldset